providers/base/bin/lid_close_suspend_open.sh

```diff
@@ -1,4 +1,19 @@
-#! /usr/bin/bash
+#!/usr/bin/bash
+
+holdoff_timeout_usec=$(gdbus introspect --system --dest org.freedesktop.login1 --object-path /org/freedesktop/login1 -p | grep HoldoffTimeoutUSec | awk '{print $5}' | awk -F\; '{print $1}')
+holdoff_timeout_sec=$(echo "scale=0; $holdoff_timeout_usec / 1000000" | bc)
```

Comment:
Bash comes with [arithmetic expansion feature](https://www.gnu.org/software/bash/manual/html_node/Arithmetic-Expansion.html) to work with numbers, so I think you should be able to use this instead:

```suggestion
holdoff_timeout_sec=$(($holdoff_timeout_usec / 1000000))
```

Reply:
This works. But an error will be reported:
[Line 4:](javascript:setPosition(4, 24))
holdoff_timeout_sec=$(($holdoff_timeout_usec / 1000000))
                       ^-- [SC2004](https://www.shellcheck.net/wiki/SC2004) (style): $/${} is unnecessary on arithmetic variables.
So remove the $ before it.